app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+def copy_file(command: str) -> None:
+    splited = command.split(" ")
+    if len(splited) != 3 or splited[0] != "cp":
+        return None
```

Comment:
Returning `None` here is unnecessary since the function's return type is `None`. Consider using `return` without a value for clarity.